miniscope_io/stream_daq.py

```diff
@@ -247,6 +260,9 @@ def _fpga_recv(
                 serial_buffer_queue.put(cur_buffer[buf_start:buf_stop].tobytes())
             if pre_pos:
                 cur_buffer = cur_buffer[pre_pos[-1] :]
+            if self.config.mode == 'RAW_REPLAY':
+                time.sleep(1) # Non-ideal. Wondering if there's a better way than checking all the queues are empty with a flag or plugging in all queues in this function.
```

Comment:
we want to make this whole chain of queues robust enough that we don't need to sleep here :). for testing purposes, we want to use the exact code that would be used normally and mock the hardware part, bc otherwise we aren't really testing the functionality of the class, but the functionality of the test conditions